lib/icalendar/calendar.rb

```diff
@@ -5,6 +5,8 @@ class Calendar < Component
     required_property :prodid
     optional_single_property :calscale
     optional_single_property :ip_method
+    optional_single_property :x_wr_calname
+    optional_single_property :x_wr_timezone
```

Comment:
Specifying 'X-' properties is unnecessary. All 'X-' prefixed properties already exist via `method_missing`